src/helpers/rateLimit.ts

```diff
@@ -42,3 +48,26 @@ export default rateLimit({
       })
     : undefined
 });
+
+export async function duplicateRequestLimit(req: Request, res: Response, next: NextFunction) {
```

Comment:
Could we move this to a separate file? It is defn. not related to the rate-limit 😄